lib/kb_SetUtilities/kb_SetUtilitiesImpl.py

```diff
@@ -131,6 +136,25 @@ def get_obj_info (self, obj_ref, obj_type_desc, full_type=False):
         (obj_name, obj_type) = self.get_obj_name_and_type_from_obj_info (obj_info, full_type)
         return (obj_info, obj_name, obj_type)
     
+    def get_newest_obj_info (self, old_obj_ref, obj_type_desc, full_type=False):
```

Comment:
It make sense to move some of this logic into a separate module so the Impl is less busy.  I think it is fine to note that for the future though.

Reply:
I agree, but it'll have to be the next iteration / refactor.  I'll add a note.